tv_apps/smart-system/js/homescreen_window_manager.js

```diff
@@ -137,11 +137,12 @@
             //
             // In this case, HomescreenWindowManager will not close and reset
             // _activeHome. We should call closeHomeApp to reset the variable
-            // and focus back.
-            this.closeHomeApp();
+            // and focus back. -> switchApp in app window manager will close it.
+            // this.closeHomeApp();
```

Comment:
Remove `// this.closeHomeApp();` and ` ->`.

At first, I thought that we should keep all homescreen related inside of this file. So, I close the home app here. But it may be too eager to close the home. This version looks better that we only call close once.

Once you remove this, you have to add `appCurrent.close()` to `switchApp`, line 198. Or you will create a regression.